org.kuchelmeister.ToolboxTuner.json

```
@@ -0,0 +1,61 @@
+{
+    "id" : "org.kuchelmeister.ToolboxTuner.Devel",
```

```suggestion
    "id" : "org.kuchelmeister.ToolboxTuner.Devel",
```

this doesn't match. `Devel` isn't a proper app name.

---

This was an accidental copy paste leftover from the development config. I fixed it in 77bcfab7c6b